gcloud/datastore/connection.py

```diff
@@ -317,9 +304,9 @@ def commit(self, project, request_pb):
         request_pb.project_id = project
         try:
             return self._stub.Commit(request_pb, GRPC_TIMEOUT_SECONDS)
-        except AbortionError as exc:
-            if exc.code == StatusCode.ABORTED:
-                raise Conflict(exc.details)
+        except _Rendezvous as exc:
```

Comment:
It's still wrong to assume properties not on the base class being caught. @tseaver Am I the only crazy one?

@nathanielmanistaatgoogle FWIW I'm confident that this would work, I'm just saying it is not very user-friendly.